include/stl2/detail/range/concepts.hpp

```diff
@@ -108,15 +108,14 @@ STL2_OPEN_NAMESPACE {
 	template <class T>
 	struct __view_predicate2 : __view_predicate3<T> {};
 
-	template <class T>
-		requires meta::_v<std::is_base_of<view_base, T>>
+	template <DerivedFrom<view_base> T>
```

Comment:
Doesn't this require "publicly and unambiguously"? I don't think we want that.

Reply:
Yes it does, and intentionally: `std::tuple<int, double, iota_view<int, int>>` is NOT a view.